drivers/builtin/include/mbedtls/private/crypto_adjust_config_tweak_builtins.h

```diff
@@ -1,28 +1,21 @@
 /**
- * \file mbedtls/config_adjust_legacy_crypto.h
- * \brief Adjust legacy configuration configuration
+ * \file mbedtls/private/crypto_adjust_config_tweak_builtins.h
```

Comment:
config_adjust_legacy_crypto.h is still referenced in:
`include/tf-psa-crypto/private/crypto_adjust_config_derived.h`
`core/tf_psa_crypto_check_config.h`
`drivers/builtin/include/mbedtls/private/config_adjust_test_accelerators.h`